spec.html

```diff
@@ -33169,7 +33168,7 @@
 
       <!-- es6num="24.2.2.1" -->
       <emu-clause id="sec-dataview-buffer-byteoffset-bytelength">
-        <h1>DataView (_buffer_ [ , _byteOffset_ [ , _byteLength_ ] ] )</h1>
+        <h1>DataView (_buffer_, _byteOffset_, _byteLength_ )</h1>
```

Comment:
yup, please ignore. github's diff randomly and silently elides sections when the file is large :-(